Remotion/Security/Core/Metadata/EnumerationReflector.cs

```diff
@@ -52,7 +52,8 @@ public EnumerationReflector ()
       Dictionary<Enum, EnumValueInfo> enumValueInfos = new Dictionary<Enum, EnumValueInfo> ();
       for (int i = 0; i < values.Count; i++)
       {
-        Enum value = (Enum) values[i];
+        // TODO: Add notnull assertion
```

Comment:
TODO-solving now or with ticket? (see all other TODOs)

Reply:
I would add tickets, just wanted to know if you agree with the todos first.

Reply:
okay, I'll check